examples/nat_wiring.json

```diff
@@ -7,6 +7,7 @@
         "next_outside_hop_mac" : "00:00:00:00:00:09",
         "tcp_start_port" : "35000",
         "udp_start_port" : "35000",
+        "tcp_time_wait" : "00:00:05",
```

Comment:
Not yet, but I plan to add scripted performance tests for throughput and timing. Obviously these are limited when run in Mininet, but would certainly be more intense than the current tests.